docs/docs/reference/release-notes/v3.0/index.md

```diff
@@ -0,0 +1,271 @@
+
+If you rely on Gatsby's default ESLint configuration you should have a smooth transition. We upgraded the underlying rules so you might get some new warnings/errors.
+
+Gatsby no longer uses the deprecated `eslint-loader`, we’ve moved to `eslint-webpack-plugin`. By using the plugin, behaviour has changed a little bit as warnings and errors are displayed later than in version 2.
+
+## gatsby-plugin-image@1.0.0
+
+This release includes a new plugin, `gatsby-plugin-image`, replacing the old `gatsby-image` plugin. Built from the ground up for speed and best practices, it's the most performant way to use images in React. If you saw your scores drop with recent Lighthouse updates, migrating to this plugin should help you get top scores again. It comes with a new, simplified GraphQL API (no more fragments), as well as a new `StaticImage` component that's as easy to use as an `<img>` tag.
+
+The `GatsbyImage` component supports three responsive layout types, as shown in the video below.
+
+<video controls autoplay loop>
+  <source
+    type="video/mp4"
+    src="/docs/reference/built-in-components/layouts.mp4"
+  />
+  <p>Your browser does not support the video element.</p>
+</video>
+
+There is a codemod available to help migrate from `gatsby-image`. Support is included in all sources that use `childImageSharp`, as well as out of the box support in several CMS plugins, with more coming soon.
+
+For more information, see:
+
+- [Migrating from gatsby-image to gatsby-plugin-image](/docs/reference/release-notes/image-migration-guide/)
+- [The step-by-step guide to using gatsby-plugin-image](/docs/how-to/images-and-media/using-gatsby-plugin-image/)
+- [The reference guide](/docs/reference/built-in-components/gatsby-plugin-image/)
+- [The plugin README](/plugins/gatsby-plugin-image/)
+
+## gatsby-source-wordpress@5.0.0
+
+Recently we’ve announced the brand new WordPress integration. Refer to [this blog post](/blog/wordpress-integration/)
+for full details.
+
+The originally published version of renewed `gatsby-source-wordpress` is `4.0.0`. It is
+fully compatible with Gatsby v2.
+
+For Gatsby v3 bump we’ve also bumped `gatsby-source-wordpress` to `5.0.0`. It should
+be a straight-forward update from `gatsby-source-wordpress@^4.0.0`, no additional changes
+from you are required.
+
+- Fixed a bug where adding a post type named "Filter" would break options queries [PR #29718](https://github.com/gatsbyjs/gatsby/pull/29718)
+- Fixed an html field gatsby-image regression that would prevent image transforms in certain situations [PR #29778](https://github.com/gatsbyjs/gatsby/pull/29778) and [PR #29813](https://github.com/gatsbyjs/gatsby/pull/29813)
+
+## gatsby-source-contentful@5.0.0
+
+- Migrated to the latest Contentful SDK, via [PR #29520](https://github.com/gatsbyjs/gatsby/pull/29520)
+- Compatibility with `gatsby-plugin-image`
+- Retries when downloading assets
+- Retries on network errors
```

Comment:
Does this need any additional context? It seems like all the other sections have used complete sentences so far